packages/core/src/list-box/ListBox.tsx

```diff
@@ -0,0 +1,265 @@
+import {
+  ComponentPropsWithoutRef,
+  FocusEvent,
+  ForwardedRef,
+  forwardRef,
+  KeyboardEvent,
+  ReactNode,
+  Ref,
+  useRef,
+} from "react";
+import { useWindow } from "@salt-ds/window";
+import { useComponentCssInjection } from "@salt-ds/styles";
+import { clsx } from "clsx";
+import { makePrefixer, useForkRef } from "../utils";
+import { defaultValueToString } from "../list-control/ListControlState";
+import { ListControlContext } from "../list-control/ListControlContext";
+import {
+  useListControl,
+  ListControlProps,
+} from "../list-control/ListControlState";
+
+import listBoxCss from "./ListBox.css";
+
+export type ListBoxProps<Item = string> = {
+  /**
+   * If `true`, the list box will be disabled.
+   */
+  disabled?: boolean;
+  /**
+   * If `true`, the list box will have not a border.
+   */
+  borderless?: boolean;
```

Comment:
Small question, are we going to make this the only component do `borderless`, v.s. form controls doing `bordered`

Reply:
It feels like a design difference, Lists by default have borders. We could always have bordered={false} but then we have to change our API standard